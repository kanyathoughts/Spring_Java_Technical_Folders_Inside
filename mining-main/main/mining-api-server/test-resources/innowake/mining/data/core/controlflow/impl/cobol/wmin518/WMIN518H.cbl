       IDENTIFICATION DIVISION.
       PROGRAM-ID. WMIN518H.
       DATA DIVISION.
       WORKING-STORAGE SECTION.
       PROCEDURE DIVISION.
           PERFORM TESTPARA
           GOBACK.
       TESTPARA.
            READ RECORDDUMMY NEXT
            AT END
            DISPLAY 'A'
            NOT AT END
            DISPLAY 'B'
            END-READ.
       UNREACHABLE.
            DISPLAY 'UNREACHABLE'
