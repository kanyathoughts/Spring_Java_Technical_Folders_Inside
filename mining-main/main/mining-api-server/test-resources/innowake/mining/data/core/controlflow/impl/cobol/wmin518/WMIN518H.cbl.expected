#5 ProcedureDivision
 #6 CobolPerformStmt TESTPARA, FROM={TestEntryPoint}, TO={#9}
  #7 CobolFieldReference
 #8 CobolGoBackStmt, FROM={#14, #18}, TO={TestReturnPoint}
 #9 CobolLabelStmt TESTPARA , FROM={#6}, TO={#10}
  #10 CobolReadStmt <branch stmt>, Label: READ RECORDDUMMY NEXT AT END DISPLAY 'A' NOT AT END DISPLAY 'B' END-READ, FROM={#9}, TO={#14(AT END), #18(NOT AT END)}
   #13 CobolAtEndBlock
    #14 CobolDisplayStmt 'A', FROM={#10(AT END)}, TO={#8}
   #17 CobolAtEndBlock
    #18 CobolDisplayStmt 'B', FROM={#10(NOT AT END)}, TO={#8}
 #21 CobolLabelStmt UNREACHABLE 
  #22 CobolDisplayStmt 'UNREACHABLE'
